test/fixtures/teams.yml

```diff
@@ -6,93 +6,107 @@ team_one:
   team_captain: user_one
   eligible: false
   division: high_school
+  users_count: 1
 
 team_two:
   team_name: team_two
   affiliation: school2
   eligible: false
   division: college
+  users_count: 0
 
 team_three:
   team_name: team_three
   affiliation: school3
   team_captain: user_four
   eligible: true
   division: professional
+  users_count: 3
 
 full_team:
   team_name: full_team
   affiliation: school3
   team_captain: full_team_user_one
   eligible: true
   division: high_school
+  users_count: 5
 
 team_with_special_chars:
   team_name: $0m3Thing @mazing
   affiliation: school!
   team_captain: user_on_team_with_special_chars
   eligible: true
   division: high_school
+  users_count: 1
 
 last_team:
   team_name: Last Team
   affiliation: lastschool
   team_captain: user_on_last_team
   eligible: true
   division: high_school
+  users_count: 1
 
 team_four:
   team_name: team_four
   affiliation: school4
   team_captain: user_four
```

Comment:
user_four is the team_captain for teams team_four through team_ten but is not a user on any of the teams.  user_four is actually a member of team_three.  team_eleven has no users nor a team_captain as there is no user_eleven.  I don't think we should remove the team_captain declarations since UserTest and UsersControllerTest both rely heavily on accessing the team_captain.  We could manually set the team_captain or add a user to trigger setting the team_captain, like in test_team_without_team_captain_is_automatically_assigned_to_first_user in TeamTest, but that would cause the same problems for team_captain as we were having with users_count